mappings/net/minecraft/util/ArraySet.mapping

```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/class_4706 net/minecraft/util/ArraySet
+	CLASS class_4707 Iterator
+		FIELD field_21566 start I
+		FIELD field_21567 index I
+	FIELD field_21562 sortingFunction Ljava/util/Comparator;
+	FIELD field_21563 entries [Ljava/lang/Object;
+	FIELD field_21564 capacity I
+	METHOD <init> (ILjava/util/Comparator;)V
+		ARG 0 initialCapacity
+		ARG 1 sortingFunction
+	METHOD method_23859 create (I)Lnet/minecraft/class_4706;
+		ARG 0 capacity
+	METHOD method_23862 addIfNotPresent (Ljava/lang/Object;)Ljava/lang/Object;
```

Comment:
```suggestion
	METHOD method_23862 addIfAbsent (Ljava/lang/Object;)Ljava/lang/Object;
```
Also, what is the returned value?

Reply:
Returned value is the what it ends up mapped as. Either the value already present, or the passed value after being inserted.